umu/reaper

```diff
@@ -1,5 +1,7 @@
 #!/usr/bin/env sh
 # Fake reaper file to avoid errors while running from source
-args="$*"
-cmd=${args#* -- }
-$cmd
\ No newline at end of file
+while [ "$1" != "--" ]; do
+  shift
+done
+shift
+"$@"
```

Comment:
This file should **only** be used in the build process so I think this should return an exit code. That way, the launcher can capture it and handle it accordingly

Reply:
The intended use is to strip the reaper part of the command so we can run from source. For me that is useful when debugging through pycharm for example. I cannot see how that related to the build process though.

If you mean to make it just echo the command for example, print the environement and similar things to aid with debugging without actually running anything, I think that is a good idea and we can do that instead.

If you mean something different, doesn't the shell return the exit code of the last command? I am unsure about it, but if it doesn't, then I'll update it to do so.

Reply:
Sorry, I was completely misunderstood of the purpose of this file then.

> The intended use is to strip the reaper part of the command so we can run from source. 

No. I'm actually of the opinion that this file should be considered to be permanently deleted to not encourage the practice of running umu from source, even for testing purposes.

But given the severity that this bug can cause, right now, I'll leave that for later or until more feedback that we really want to allow this. But again, I never think we should.